tests/ts_simple_query_buffers_SUITE.erl

```diff
@@ -393,3 +392,10 @@ col_no({[A|_], _, _}) ->
 
 fmt(F, A) ->
     lists:flatten(io_lib:format(F, A)).
+
+get_write_perm_cmd(take_away, QBufDir) ->
+    ct:pal("take away perms on ~s\n", [QBufDir]),
+    fmt("sh -c \"mv '~s' '~s.boo' && touch '~s'\"", [QBufDir, QBufDir, QBufDir]);
```

Comment:
Rather than using the shell, what about native Erlang like `file:rename/2` and `file:write/2`?  Or back to the original permission solution with `file:change_mode/2`?